Orange/widgets/classify/tests/test_owknnclassification.py

```diff
@@ -1,53 +1,30 @@
 # Test methods with long descriptive names can omit docstrings
 # pylint: disable=missing-docstring
-from PyQt4 import QtGui
-
-from Orange.widgets.tests.base import WidgetTest
 from Orange.widgets.classify.owknn import OWKNNLearner
+from Orange.widgets.tests.base import (WidgetTest, WidgetLearnerTestMixin,
+                                       GuiToParam)
 
 
-class TestOwKnnClassification(WidgetTest):
+class TestOWKNN(WidgetTest, WidgetLearnerTestMixin):
```

Comment:
If the test case name matches TestClassName syntax, PyCharm will allow you to jump between tests and code using a keyboard shortcut (CMD + T on mac)